app/view/state/__init__.py

```diff
@@ -29,9 +29,11 @@ def register():
     result=state.create(name, abbreviation, capital,population, year_admitted)
 
     if result:
-        return jsonify({"message": "state created successfully", "status": 201,"result":result})
+        response=make_response({"message": "state created successfully", "status": 201,"result":data},201)
+        return response
```

Comment:
kindly expound more on this comment

Reply:
What I mean is that instead of having to assign `make_response(dict, status code)` to a variable `reponse`, why not just return it. 
e.g
`return make_response({"message": "state created successfully", "status": 201,"result":data},201)`